var/spack/repos/builtin/packages/py-amici/package.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyAmici(PythonPackage):
+    """Advanced Multilanguage Interface for CVODES and IDAS"""
+
+    homepage = "https://github.com/AMICI-dev/AMICI"
+    pypi = "amici/amici-0.11.28.tar.gz"
+
+    version("0.16.0", sha256="1a2d6633ec34241d8d8b496d18d4318482cffe125e9ddf3ca6cac5d36d235f38")
+    version("0.11.28", sha256="a8ddda70d8ebdc40600b4ad2ea02eb26e765ca0e594b957f61866b8c84255d5b")
+
+    variant("boost", default=False, description="Enable boost support")
+    variant("hdf5", default=False, description="Enable HDF5 support")
+
+    depends_on("py-setuptools@48:", type="build")
```

Comment:
```suggestion
    depends_on("py-setuptools@48:", type=("build", "run"))
```

Reply:
So I know what to look out for in the future: why is it a run dependency as well this time?

Reply:
https://spack.readthedocs.io/en/latest/build_systems/pythonpackage.html#setuptools

It's listed under `install_requires`, and is imported in the code that gets installed.